### DISCUSSION TITLE
Can I hide "UP NEXT" in fullscreen?

### DISCUSSION THREAD

**Discussion by Timtaran** [daksh2k/Spicetify-stuff#159] [Q&A] [answered]
.

**Comment by daksh2k** [comment, marked as answer]
You can toggle it in settings, open it by right clicking anywhere @Timtaran 
<img width="600" alt="image" src="https://github.com/daksh2k/Spicetify-stuff/assets/42179185/ef362478-e03c-4599-b729-ea2881c83c47">
